src/components/Home/Home.jsx

```diff
@@ -1,7 +1,14 @@
+import './Home.css'
+
 const Home = () => { 
 
-  <h>This is the Home Page</h>
-  console.log('Home page works')
+  return (
+    <>
+  <h1 className='welcome-text'>Welcome to the Home Page</h1>
+
```

Comment:
It might be nice for the user to see some image on the home page. Take a look at some news websites for inspiration! 😄